docs/coding-guidelines/project-guidelines.md

```diff
@@ -244,7 +244,7 @@ Each source file should use the following guidelines
   - `.WinRT.cs` - implementation based on [WinRT](https://en.wikipedia.org/wiki/Windows_Runtime)
 
 ## Define naming convention
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).

```suggestion
## Define naming convention

```